builder.json

```
@@ -78,6 +78,27 @@
                         "-DBUILD_TESTING=OFF"
                     ]
                 }
+            },
+            "hosts": {
+                "alpine": {
```

suggestion: add a comment explaining why this is necessary
```suggestion
                "_comment": "???????what is going on here?????",
                "alpine": {
```

---

Thanks, I have added a comment.